tests/webdav-jest.ts

```diff
@@ -3,6 +3,7 @@ import NextcloudClient    from '../source/client';
 import configuration      from './configuration';
 import * as Stream        from 'stream';
 import { Request }        from 'request';
+import * as Path          from 'path';
```

Comment:
OK, this isn't a front-end library, but couldn't we tree-shake by importing `join` only? It seems to be the only method used throughout the file.